README.md

```diff
@@ -50,7 +50,7 @@ If you are using Maven without the BOM, add this to your dependencies:
 If you are using Gradle 5.x or later, add this to your dependencies:
 
 ```Groovy
-implementation platform('com.google.cloud:libraries-bom:26.36.0')
+implementation platform('com.google.cloud:libraries-bom:26.37.0')
```

Comment:
This update is already handled in https://github.com/googleapis/java-bigquerystorage/pull/2459